package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "@cospired/i18n-iso-languages": "^2.0.2",
     "@edx/edx-bootstrap": "git://github.com/edx/edx-bootstrap.git#update-with-documentation-site",
+    "@edx/frontend-analytics": "file:../frontend-analytics",
```

Comment:
Reviewers: This will be updated before merge.

Reply:
Related to this, maybe, are those 15k lines added to package-lock.json... something seems wrong there. Looking at the definition of `@edx/frontend-auth` in that file, it's only like a dozen lines long, whereas the definition for frontend-analytics is... 15k lines. :)

Reply:
Yes.  Should be all set when I can switch to the real package.

Reply:
FYI: this is all set.